src/scripts/extensions/bookmarklet/bookmarkletInject.ts

```diff
@@ -1,4 +1,3 @@
-declare var frameUrl: string;
```

Comment:
Do we know the implications of removing frameUrl from this file? Bookmarklet is another client type and it would be good to test the conditions when this gets invoked.